test/e2e/sagemaker/resources/endpoint.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Endpoint
+metadata:
+  name: $ENDPOINT_NAME
+spec:
+  endpointName: $ENDPOINT_NAME
+  endpointConfigName: $ENDPOINT_NAME-config
```

Comment:
This seems to couple the names of endpoints in a way that I could see being unclear, because you have to look into the yaml and find this. It would probably be a lot more self-explanatory if there was `ENDPOINT_NAME` and `ENDPOINT_CONFIG_NAME`. How these are related can be in the Python code.